vagrant/provisioning/roles/arkcase-ldap-config/templates/spring-config-ldap.properties

```diff
@@ -32,8 +32,8 @@ ldapConfig.changedGroupSearchFilter=(&(objectclass=group)(modifyTimestamp>=%s))
 ldapConfig.groupSearchPageFilter=(&(objectclass=group)(cn>=%s))
 ldapConfig.changedGroupSearchPageFilter=(&(objectclass=group)(cn>=%s)(modifyTimestamp>=%s))
 ldapConfig.groupsSortingAttribute=cn
-ldapConfig.userPrefix={{ ldap_prefix }}
-ldapConfig.groupPrefix={{ ldap_prefix }}
+ldapConfig.userPrefix={{ ldap_prefix }}{{ portal_prefix }}
+ldapConfig.groupPrefix={{ ldap_prefix }}{{ portal_prefix }}
```

Comment:
The portal_prefix for the groups should be empty string. So, I think we should have separate variables for user and groups. We do have separate properties as well. So we should have:

- ldap_user_prefix
- ldap_group_prefix
- ldap_portal_user_prefix
- ldap_portal_group_prefix

Reply:
Probably is better this way.

Reply:
I just won't include '{{ portal_prefix }}' on the ldapConfig.groupPrefix; then the group prefix will never have a portal suffix, and no chance of group prefix and user prefix being out of line.  Way too confusing to have different user prefix and group prefix in the same environment.

Reply:
@bojmil-armedia , @sasko-tanaskoski, change applied

Reply:
I'm good with this.